RN 40877 (#3887)
SkylineCommunications/dataminer-docs: user-guide/Basic_Functionality/Elements/Working_with_elements/Adding_Elements/HTTPS_Connection.md

  - To disable certificate validation for an element created after a 10.4.12/10.5.0 upgrade, enable the *Insecure HTTPS* option.

  - To enable certificate validation for an HTTP element created before a 10.4.12/10.5.0 upgrade, disable the *Insecure HTTPS* option.

  When the *Insecure HTTPS* setting is enabled, in case an HTTP connector polls an HTTPS endpoint:

  - DataMiner will ignore invalid certificates in the following cases:

    - The server certificate is expired.

    - The server certificate is revoked.

    - The common name of the server certificate does not match the server name to which DataMiner is sending the request.

    - The certificate is issued by a Certificate Authority that is not trusted by the DataMiner Agent.

  - DataMiner will block communication in the following cases:

    - The server is offering a non-server certificate.

    - The server certificate is signed by a weak signature.

  > [!NOTE]
  >
  > - To skip SSL/TLS certificate validation for all elements that share the same *protocol.xml* file, set the `InsecureHttps` element to true in the `PortSettings` element of the *protocol.xml* file.

>   > - To skip SSL/TLS certificate validation when using multi-threaded HTTP communication, set `requestSettings[6]` to true when building the HTTP request in a QAction. For more information, see [Setting up multi-threaded HTTP communication in a QAction](xref:AdvancedMultiThreadedTimersHttp).

Author: @PedroDebevere, it seems that the `InsecureHttps` element might need to be added to [this page](https://docs.dataminer.services/develop/schemadoc/Protocol/Protocol.PortSettings.html) in the development guide, and it may also benefit from its own subpage. Additionally, [this page about setting up multi-threaded HTTP communication](https://docs.dataminer.services/develop/devguide/Connector/AdvancedMultithreadedTimersHttp.html) might need an update to include `requestSettings[6]`. 

Could you create a pull request for this? Thanks in advance!

Reviewer: @EdithLansens Since the feature has been merged, it has been decided that this will be renamed to "SkipCertificateVerification" and it is a blocking change for the release.
This change is currently in code review, the associated release note is 41285.

Reviewer 2: PR created: #3916

- **Network**: The network interface (NIC). If only one network interface is available on the DMA, it is automatically selected.

> - **Skip SSL/TLS certificate verification (insecure)**: From DataMiner 10.4.12/10.5.0 onwards<!--RN 40877 + 41285-->, SSL/TLS certificates are validated by default for all HTTP elements created after these versions are installed. When this setting is enabled, SSL/TLS certificates will not be automatically validated.

Author: ```suggestion
- **Skip SSL/TLS certificate verification (insecure)**: From DataMiner 10.4.12/10.5.0 onwards<!--RN 40877 + 41285-->, SSL/TLS certificates are validated by default for all newly created HTTP elements. When this setting is enabled, SSL/TLS certificates will not be automatically validated.
```

> 

Author: Done in [commit 0a05079](https://github.com/SkylineCommunications/dataminer-docs/pull/3887/commits/2a05079e3879bef0cf5afb91e1aecbcc72328cef). Thanks for the feedback!
